core/tauri-macros/src/lib.rs

```diff
@@ -151,3 +154,38 @@ pub fn do_menu_item(input: TokenStream) -> TokenStream {
   let tokens = parse_macro_input!(input as menu::DoMenuItemInput);
   menu::do_menu_item(tokens).into()
 }
+
+/// Convert a .png or .ico icon to an Image
+/// for things like `tauri::tray::TrayIconBuilder` to consume,
+/// relative paths are resolved from `CARGO_MANIFEST_DIR`, not current file
+///
+/// ### Examples
+///
+/// ```ignore
+/// TrayIconBuilder::new().icon(include_image!("./icons/32x32.png")).build().unwrap();
```

Comment:
I think we need to showcase different use cases for it, maybe something like this

```suggestion
/// const APP_ICON: Image<'_> = include_image!("./icons/32x32.png");
/// 
/// // then use it with tray
/// TrayIconBuilder::new().icon(APP_ICON).build().unwrap();
///
/// // or with window
/// WebviewWindowBuilder::new(app, "main", WebviewUrl::default())
///     .icon(APP_ICON)
///     .unwrap()
///     .build()
///     .unwrap();
///
/// // or with any other functions that takes `Image` struct
```